Karumi/Shot
# Add parameter "onlyFailing", which will prevent output of successful tests

Fix https://github.com/Karumi/Shot/issues/175

### :pushpin: References
* **Issue:** https://github.com/Karumi/Shot/issues/175

### :tophat: What is the goal?
Have additional parameter to only output failing tests, to reduce output on CI

### How is it being implemented?
Optional parameter

### How can it be tested?
- have a failing test
- run it normally, see e.g. 11 successful, 1 failing in comparison
- run it with -PonlyFailing, see only 1 failing in table (headline remains)
